### DISCUSSION TITLE
PnP - Search Filter are not working as exepcted. FileName will show the extension without dot + LastModifiedDateTime will now show any results

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2802] [Q&A]
I have defined the following inside my PnP Search - Filters, to filter based on FileName & LastModifiedDatetime:-

![image](https://user-images.githubusercontent.com/79726511/221051932-01854f73-184a-47b5-997b-c79555f4cfff.png)

But i am facing those 2 issues:-

1) fileName will show options as follow, where the filename will be combined with the file extension and without dot between them as follow:-

![image](https://user-images.githubusercontent.com/79726511/221052225-73050260-5546-4c2e-9728-c5652d5d958a.png)

2) Defining Resource.LastModifiedDateTime will not show any results for any time intervals, as follow:-

![image](https://user-images.githubusercontent.com/79726511/221052469-b35a2753-9b55-4b0b-b34d-c9faa6b9eca0.png)

any advice on why the 2 filters are not working?

Thanks

**Comment by wobba** [comment]
Can you use the debug template and inspect the values coming back? Looks like you are using the Graph connector so you should check the data coming back first.

**Reply by mvcsharepointdev** [reply]
@wobba what i need to do exactly? did not get your point, i am configuring the built-in capabilities inside the PnP Modern search result

**Reply by wobba** [reply]
Look at the returned data using the debug template. This shows the objects and structure and values returned.

![image](https://user-images.githubusercontent.com/51104/221957029-55b8c229-184c-4908-9547-c0b3f1be642c.png)
